DQM/HLTEvF/python/HLTSiStripMonitoring_cff.py

```diff
@@ -36,15 +35,13 @@
   size_cutX = cms.double( 3.0 ),
   inflate_all_errors_no_trk_angle = cms.bool( False ),
   IrradiationBiasCorrection = cms.bool( False ),
```

Comment:
@mtosi the configuration of these producers looks somewhat outdated to me. I only fixed the patently useless parameters, though it would be good to revise what is done in the Tracking HLT DQM. I guess this can be left for a follow-up PR.